examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2351,45 +2354,44 @@ class LDrawLoader extends Loader {
 
 		}
 
+		material.color.setStyle( color, COLOR_SPACE_LDRAW );
```

Comment:
Should `color` be named 'hexColor' for clarity?

Reply:
I'm not sure on this, have clarified fillColor / edgeColor for now.

Reply:
Just not `color`.  :-)